docs/how-to/release_process/validation_pipeline_execution.rst

```diff
@@ -0,0 +1,103 @@
+Validation pipeline execution
+=============================
+
+When validating various versions of Checkbox multiple workflows are in play.
+Everything starts with automated creation of an Edge version.
+
+Creating new Edge version
+-------------------------
+Every day github checks for new commits that landed in the Checkbox repository,
+and if anything new landed a new version is created.
+
+
+.. mermaid::
+
+    graph TB
+
+        A[GitHub Workflow: Detect New Commits]
+
+        B{Run Metabox Tests on New Version}
+        C[Mark Version as Edge and Build Snaps]
+        D{Did Metabox Tests Pass?}
+        E{Did Snap Builds Succeed?}
+        F[Upload New Edge Version to Store]
+
+        G[End: Build Failed - No New Edge Version]
+
+
+        A --> B
+        B --> D
+        D -->|Yes| C
+        C --> E
+        E -->|Yes| F
+        D -->|No| G
+        E -->|No| G
+
+With new Edge version of Checkbox in the store we can start validating it.
+
+Validating the Edge version
+---------------------------
+
+On the Cert Jenkins instance, the `{insert_final_job_name_here}` checks the store API for new Edge versions of Checkbox.
+The job is defined in the hwcert-jenkins-jobs repository here: `{insert url to the trigger job}`.
+This job is also responsible for checking if all of the necessary snaps were published (for other series and architectures).
+Once confirmed the "Canary Test Plan" defined here: `Canry test plan <https://github.com/canonical/checkbox/blob/main/providers/base/units/canary/test-plan.pxu>`_
```

Comment:
I think the end of the sentence is missing ("is executed." or something like that).

And, as above, you can just link to that test plan instead of pasting the URL.

Reply:
Fixed in 0505d9dec